modules/custom/az_news/config/install/views.view.az_news.yml

```diff
@@ -84,7 +87,7 @@ display:
         type: 'entity:node'
         options:
           relationship: none
-          view_mode: az_media_list
+          view_mode: az_feature
```

Comment:
Are we sure we want this to update existing sites referencing the Master display to now see the az_feature view mode?

Reply:
No :( I'll commit this. Thank you!